CHANGELOG.md

```diff
@@ -31,6 +31,8 @@
   configured log target-url includes credentials. Thanks to
   [Amir M. Saeid](https://github.com/amir) for the contribution.
   [#2430](https://github.com/Mashape/kong/pull/2430)
+- Logging retries and failure information.
+  [#2429](https://github.com/Mashape/kong/pull/2429).
```

Comment:
This should have went into the below section already having a "logging plugins" mention